terraform/development/main.tf

```diff
@@ -44,3 +44,9 @@ module "database" {
   ingress_source_security_group_ids = []
   vpc_id                            = module.networking.vpc_id
 }
+
+module "service" {
+  source = "../modules/service"
+
+  prefix = "core-stag"

```

Comment:
🤦 - I almost didn't bother actually putting this in at all for dev because we'll need to re-evaluate it all come review apps anyway, obviously wasn't paying attention. (Maybe a separate chat to have about whether we should be bothering to fill in these files at the moment, but for now I'll update the prefix - and you're right it really should be a local variable)